test/integration_helper.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/ModuleLength
 module IntegrationHelper
```

Comment:
```suggestion
module IntegrationHelper # rubocop:disable Metrics/ModuleLength
```
Add it to the same line as the module definition and you don't need to disable it anymore at the end of the module definition.